00-programming-fundamentals/debugging-with-vs-code.md

```diff
@@ -169,6 +169,22 @@ Okay, so we have stepped forward, and we've seen a change right before our eyes.
 
 At this point, we would use the information we gathered during our debugging to fix the code, or, if the problem didn't arise, we might try again with new parameters.
 
+## Make it Portable
+
+So, this is very powerful functionality, but if we leave our configuration in the `launch.json` we aren't going to have access to them whenever we start a new project. As things are set up right now, we would need to copy this `launch.json` into every project, forever. 
+
+We are going to fix this by making a change to the settings in VS Code. Using `⌘,` or  the drop down menu as seen below, open the settings menu.
+
+![Drop down Code, drop down Preferences, click Settings.](images/debug_settings_1.png "Drop down Code, drop down Preferences, click Settings.")
+
+When you have opened the settings, the bar at the top will have a curly brace icon as seen below. Clicking on it will open your `settings.json` for VS Code.
```

Comment:
I think you need another step in here for them to find where to put or enter `launch.json`.  I'd put a screenshot in showing the search feature of settings.

Not pertinent to this PR, but I also see a broken image in the older content on this branch.